gnome/gtk-doc/Portfile

```diff
@@ -83,32 +83,32 @@ proc py_setup {p_py_ver} {
                     port:py${py_ver_nodot}-mock
 }
 
-variant python38 conflicts python39 python310 python311 python312 description {Build using Python 3.8} {
-    py_setup 3.8
-}
-
-variant python39 conflicts python38 python310 python311 python312 description {Build using Python 3.9} {
+variant python39 conflicts python38 python310 python311 python312 python313 description {Build using Python 3.9} {
```

Comment:
You need to remove the "python38" from all the "conflicts"

Reply:
Nice catch, done